src/client/camera.cpp

```diff
@@ -382,9 +382,9 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 tool_reload_ratio)
 			eye_offset += player->eye_offset_third;
 			break;
 		case CAMERA_MODE_THIRD_FRONT:
-			eye_offset.X += player->eye_offset_third.X;
-			eye_offset.Y += player->eye_offset_third.Y;
-			eye_offset.Z -= player->eye_offset_third.Z;
+			eye_offset.X += player->eye_offset_third_front.X;
+			eye_offset.Y += player->eye_offset_third_front.Y;
+			eye_offset.Z -= player->eye_offset_third_front.Z;
```

Comment:
I think it would be more confusing if the negation was only applied to the default value.

Imagine someone writes `player:set_eye_offset(nil, vector.new(0, 0, -5))`. Now they want the offset to be a bit smaller in third person front view, and they write `player:set_eye_offset(nil, vector.new(0, 0, -5), vector.new(0, 0, -4))` instead. With the current version of the PR, this works as expected. With your suggested change, the Z offset in third person front view would be applied reversed (relative to what I would expect).